guides/common/modules/proc_using-container-registries.adoc

```diff
@@ -30,3 +46,24 @@ Pulling container images:
 ----
 # podman pull {foreman-example-com}/my-image:<optional_tag>
 ----
+
+Pushing container images to the {Project} container registry:
+
+* To indicate which organization, product, and repository the container image belongs to, use container namespaces.
```

Comment:
"Container namespaces" isn't in the OCI container spec, so I think we can leave that term out.
```suggestion
* To indicate which organization, product, and repository the container image belongs to, include the organization and product in the container repository name.
```